test

```diff
@@ -27,7 +27,7 @@ if [ -z "$GOPATH" ]; then
 	go get -u github.com/coreos/pkg/dlopen
 fi
 
-TESTABLE="activation journal login1 machine1 unit"
+TESTABLE="activation journal login1 machine1 unit daemon"
 FORMATTABLE="$TESTABLE sdjournal dbus"
 if [ -e "/run/systemd/system/" ]; then
 	TESTABLE="${TESTABLE} sdjournal"
```

Comment:
Your test requires a running systemd, so I think it should be placed here instead (guarded by the if) and in $FORMATTABLE too.